app/models/article.rb

```diff
@@ -19,12 +19,15 @@ class Article < ApplicationRecord
   delegate :name, to: :user, prefix: true
   delegate :username, to: :user, prefix: true
 
-  belongs_to :user
-  belongs_to :organization, optional: true
   # touch: true was removed because when an article is updated, the associated collection
   # is touched along with all its articles(including this one). This causes eventually a deadlock.
   belongs_to :collection, optional: true
 
+  belongs_to :organization, optional: true
+  belongs_to :second_user, class_name: "User", inverse_of: :articles_as_second_user, optional: true
+  belongs_to :third_user, class_name: "User", inverse_of: :articles_as_third_user, optional: true
```

Comment:
added this two

Reply:
Oh hmmmmmm so I think we have a PR soon that will change this to an array column and deprecate `article.second_user_id` and `article.third_user_id`. Might be better to hold off on changing them for now?

cc @juliannatetreault who is working on refactoring the column.

Reply:
Correct, @Zhao-Andy! I have a draft PR open right now that removes `second_user_id` and `third_user_id` in favor of `co_author_ids` that should be ready for review in the _very_ near future (I ran into some major DB problems last night into this morning that held the PR up).

Reply:
Ah, didn't know :( I'll remove those from the PR then

Reply:
Thanks, sorry for the extra work!

Reply:
So, so sorry, @rhymes! I should have communicated this better. 😞

Reply:
ahha no problem :) I'll take a look at your PR when it's ready @juliannatetreault!

Reply:
Just added you as a reviewer, @rhymes! 😎